src/components/Footer/footer.css

```
@@ -60,47 +60,40 @@
     gap: 20px;
 }
 
-/* sociial media-------------------------------------------- */
+/* social media-------------------------------------------- */
 
-.fa {
+.social-icon {
     display: flex;
     justify-content: center;
     align-items: center;
-    padding: 10px;
-    font-size: 30px;
+    font-size: 40px;
+    color: #4d4d4d;
     width: 45px;
     height: 45px;
-    text-decoration: none;
-    border-radius: 50px;
     transition: 400ms;
```

```suggestion
    transition: 200ms;
```
Στα 400ms μου φαίνεται λίγο αργό το transition.